packages/@react-stately/color/src/Color.ts

```diff
@@ -266,6 +270,45 @@ class RGBColor extends Color {
   getColorSpace(): ColorFormat {
     return 'rgb';
   }
+
+  private static colorChannels: Set<ColorChannel> = new Set(['red', 'green', 'blue']);
+  getColorChannels(): Set<ColorChannel> {
+    return RGBColor.colorChannels;
+  }
+  getColorSpaceAxes({xChannel, yChannel}): ColorAxes {
+    let xCh;
+    let yCh;
+    if (!xChannel) {
+      switch (yChannel) {
+        case 'red':
+        case 'green':
+          xCh = 'blue';
+          break;
+        case 'blue':
+          xCh = 'red';
+          break;
+        default:
+          xCh = 'blue';
+          yCh = 'green';
+      }
+    } else if (!yChannel) {
+      switch (xChannel) {
+        case 'red':
+          yCh = 'green';
+          break;
+        case 'blue':
+          yCh = 'red';
+          break;
+        default:
+          xCh = 'blue';
+          yCh = 'green';
+      }
+    }
+    let xyChannels: Set<ColorChannel> = new Set([xChannel, yChannel]);
+    let zChannel = difference(this.getColorChannels(), xyChannels).values().next().value as ColorChannel;
+
+    return {xChannel: xCh, yChannel: yCh, zChannel};
+  }
```

Comment:
A runtime error will be thrown because the returned object will have `{xChannel: undefined, yChannel: undefined, zChannel}` when an `xChannel` or `yChannel` value is provided.

Either initialize local variables as `xCh = xChannel` and `yCh = yChannel` or just use `xChannel` and `yChannel` as local variables. 

```suggestion
  getColorSpaceAxes({xChannel, yChannel}): ColorAxes {
    if (!xChannel) {
      switch (yChannel) {
        case 'red':
        case 'green':
          xChannel = 'blue';
          break;
        case 'blue':
          xChannel = 'red';
          break;
        default:
          xChannel = 'blue';
          yChannel = 'green';
      }
    } else if (!yChannel) {
      switch (xChannel) {
        case 'red':
          yChannel = 'green';
          break;
        case 'blue':
          yChannel = 'red';
          break;
        default:
          xChannel = 'blue';
          yChannel = 'green';
      }
    }
    let xyChannels: Set<ColorChannel> = new Set([xChannel, yChannel]);
    let zChannel = difference(this.getColorChannels(), xyChannels).values().next().value as ColorChannel;

    return {xChannel, yChannel, zChannel};
  }
```